doc/install.md

```diff
@@ -51,7 +51,7 @@ You can also build your own executables from source code.
 
 &ensp; 2\. Make sure you have the necessary libraries installed:
 
- * Linux (Ubuntu): `apt-get update && apt-get -y install build-essential pkg-config autoconf gnutls-dev git`
+ * Linux (Ubuntu): `apt-get update && apt-get -y install build-essential pkg-config autoconf gnutls-dev git make`
```

Comment:
`make` should be installed as part of `build-essential`, so I don't think this is necessary. See https://packages.ubuntu.com/xenial/build-essential

Reply:
Good catch!

Fixed here: https://github.com/livepeer/go-livepeer/pull/1479/commits/b7df02319ea3d2262870db82fa05dbc3a7db0a26

Reply:
Thanks Chris!